src/sentry/api/endpoints/internal_ea_features.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.base import Endpoint, region_silo_endpoint
+from sentry.api.bases.organization import OrganizationAdminPermission
+from sentry.conf.server import SENTRY_EARLY_FEATURES
+from sentry.models.organization import Organization
+
+
+@region_silo_endpoint
+class InternalEAFeaturesEndpoint(Endpoint):
+    permission_classes = (OrganizationAdminPermission,)
+
+    def get(self, request):
+        features_dict = features.all()
+
+        ea_org = Organization()
+        ea_org.flags.early_adopter = True
+
+        ea_features = [
+            feature_name for feature_name in features_dict if features.has(feature_name, ea_org)
+        ]
```

Comment:
What are we using this for? You might want to use `batch_has` here so it's not extremely slow